kong/pdk/log.lua

```diff
@@ -459,20 +459,19 @@ end
 --
 -- Where:
 --
--- * `%file_src`: is the file name from where the log was called from.
--- * `%func_name`: is the name of the function from where the log was called
---   from.
--- * `%line_src`: is the line number from where the log was called from.
--- * `%message`: is the message, made of concatenated, pretty-printed arguments
+-- * `%file_src`: The filename from where the log was called.
```

Comment:
Nit: the way we've changed these makes the `Where:` on line 460 a little grammatically awkward, since the implied way of reading it to oneself was something like:

> Where `%func_name` is the file name from where the log was called from.
> and
> `%func_name` is the name of the function from where the log was called from.
> [...]

So with the new phrasing, I think it might make sense to drop the `Where:`. What do you think?

Reply:
Nice catch. This one was weird, I was struggling to phrase it in a cleaner way. How about:

> %file_src`: The filename the log was called from.

Bringing back the closing "from", but makes more sense in the phrase "Where `%file_src` [is] the filename the log was called from."

Reply:
> %file_src`: The filename the log was called from.

I like this!